units/systemd-networkd.service.in

```diff
@@ -42,6 +42,7 @@ SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
 SystemCallFilter=@system-service
 Type=notify
+RestartKillSignal=SIGUSR2
```

Comment:
Probably just bikeshedding, but I think SIGHUP would be a more conventional choice for this.

Reply:
Yes I think SIGHUP is pretty generic for this . We should do that too.

Reply:
Not really. SIGHUP is conventionally used for "configuration reload". This is about optimizing a full stop-and-start operation to skip some steps if we know that the stop will be followed by a start. In fact, if we ever add support for reloading configuration to networkd without restarting, we'd almost certainly use SIGHUP for this. We shouldn't mix those two operations.